usaspending_api/accounts/tests/test_federal_account_autocomplete.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+import pytest
+from model_mommy import mommy
+from rest_framework import status
+
+from usaspending_api.accounts.models import FederalAccount
+from usaspending_api.common.tests.autocomplete import check_autocomplete
+
+
+@pytest.fixture
+def fed_account_data(db):
+    mommy.make(
+        FederalAccount,
+        account_title='zzz',
+        main_account_code='abc')
+    mommy.make(
+        FederalAccount,
+        account_title='###',
+        main_account_code='789')
+    mommy.make(FederalAccount, main_account_code='XYZ')
+
+
+@pytest.mark.parametrize("fields,value,expected", [
```

Comment:
You've been doing them all along, but just wanted to say again how much I love these parameterized tests courtesy of pytest